DOCS-611 - Add SSO content for organizations (#5081)
SumoLogic/sumologic-documentation: docs/manage/manage-subscription/create-and-manage-orgs/manage-org-settings.md

1. Enter a new subdomain name. The name must be between 4 and 63 characters in length at least four characters in length, and can contain lower case letters, numbers, and dashes only. <br/><img src={useBaseUrl('img/manage/subscriptions/change-subdomain-name.png')} style={{border: '1px solid gray'}} alt="change-subdomain-name.png" width="700" />
1. Click **Change Subdomain** Name to update the name.
1. You will be automatically logged out and redirected to the new subdomain login page. 

## SSO for child organizations

When you create a new child organization, a subdomain is automatically created and single sign-on (SSO) is enabled, allowing you to sign in to the child organization without having to provide separate credentials.

### Enabling or disabling SSO

#### Enable SSO

> If a subdomain exists, but SSO is disabled for the child organization, click the **Enable SSO** button:<br/><img src={useBaseUrl('img/manage/subscriptions/enable-sso-button.png')} alt="Enable SSO button" style={{border: '1px solid gray'}} width="250"/>

Reviewer: ```suggestion
If a subdomain exists, but SSO is disabled for the child organization, click **Enable SSO**:<br/><img src={useBaseUrl('img/manage/subscriptions/enable-sso-button.png')} alt="Enable SSO button" style={{border: '1px solid gray'}} width="250"/>
```

#### Disable SSO

When a child organization has SSO enabled, the child organization's details display the subdomain URL in the **Subdomain** field, and the **SSO** field says **Enabled**:<br/><img src={useBaseUrl('img/manage/subscriptions/subdomain-and-sso-enabled.png')} alt="SSO enabled for a child organization" style={{border: '1px solid gray'}} width="250"/>

If you want to require administrators to enter credentials to sign in to the child organization, click **Disable SSO** to turn off single sign-on.

#### Create a subdomain

If there is no subdomain defined for a child organization, hovering your mouse over the **Enable SSO** button displays the message **Set up custom subdomain before enabling SSO**:<br/><img src={useBaseUrl('img/manage/subscriptions/no-subdomain-defined.png')} alt="No subdomain defined" style={{border: '1px solid gray'}} width="300"/>

To set up a custom subdomain for the child organization, see [Set up a custom subdomain](#set-up-a-customsubdomain) above. Once you set up the subdomain, the subdomain's URL will appear and the **Enable SSO** button will be clickable.

### Sign in to a child organization automatically

> When you open the details of a child organization, a link for the child organization appears under **Basic Details**. When you click the link, you are automatically signed in to the child organization. 

Reviewer: ```suggestion
When you open the details of a child organization, a link to the child organization appears under **Basic Details**. When you click the link, you are automatically signed in to the child organization. 
```

> <img src={useBaseUrl('img/manage/subscriptions/mssp-sso-enabled.png')} alt="SSO enabled for a child organization" style={{border: '1px solid gray'}} width="300"/>

Reviewer: Your email address is exposed in the image fyi

Author: Yikes!

> Automatic sign-in works because when you created the child organization, a [custom subdomain](#set-up-a-customsubdomain) was automatically added, and SSO was enabled by default. Therefore, you are already provisioned as a user in the child organization and can access the organization at any time without needing to log in.

Reviewer: ```suggestion
Automatic sign-in works because when you created the child organization, a [custom subdomain](#set-up-a-customsubdomain) was automatically added, and SSO was enabled by default. As a result, you are already provisioned as a user in the child organization and can access it at any time without needing to log in.
```

> ### Sign back in with parent org credentials

Reviewer: For consistency, should org be organization?

Author: Yes. Good point.

> As an administrator, if you log out of a child organization that has SSO enabled, the following screen appears. Click the **Login with Parent Org** button to automatically log in using your credentials from the parent organization.

Reviewer: ```suggestion
As an administrator, if you log out of a child organization with SSO enabled, the following screen appears. Click **Login with Parent Org** to sign back in automatically using your parent organization credentials.
```
